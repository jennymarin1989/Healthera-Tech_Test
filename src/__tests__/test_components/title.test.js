import React from 'react';
import styled from 'styled-components';
import render from 'react-test-renderer';
import 'jest-styled-components';

describe('title', () => {
    let title = render.create(<Title />).toJSON();
  it("returns title with specific style", () => {
     expect(title).toMatchSnapshot()
     expect(title).toHaveStyleRule('color', 'green') 
  })  

})