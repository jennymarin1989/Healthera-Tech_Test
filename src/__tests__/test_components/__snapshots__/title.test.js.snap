// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`title returns title with specific style 1`] = `
.c0 {
  font-size: 0.8em;
  color: MediumSeaGreen;
  font-family: verdana;
}

<text
  className="c0"
/>
`;
